config.toml

```diff
@@ -88,22 +85,28 @@ sidebar_menu_compact = true
 # 	icon = "fa fa-envelope"
 #         desc = "Discussion and help from your fellow users"
 # Developer relevant links. These will show up on right side of footer and in the community page if you have one.
-[[params.links.developer]]
+# [[params.links.developer]]
+# 	name = "GitHub"
+# 	url = "https://github.com/matrix-org"
+# 	icon = "fab fa-github"
+#   desc = "Matrix on GitHub"
+# Custom links shown in the center of the footer.
+[[params.links.bottom]]
 	name = "GitHub"
 	url = "https://github.com/matrix-org"
 	icon = "fab fa-github"
   desc = "Matrix on GitHub"
-[[params.links.developer]]
+[[params.links.bottom]]
 	name = "GitLab"
 	url = "https://gitlab.matrix.org/matrix-org"
 	icon = "fab fa-gitlab"
   desc = "Matrix on GitLab"
-[[params.links.developer]]
+[[params.links.bottom]]
 	name = "YouTube"
 	url = "https://www.youtube.com/channel/UCVFkW-chclhuyYRbmmfwt6w"
 	icon = "fab fa-youtube"
   desc = "Matrix YouTube channel"
-[[params.links.developer]]
+[[params.links.bottom]]
 	name = "Twitter"
 	url = "https://twitter.com/matrixdotorg"
 	icon = "fab fa-twitter"
```

Comment:
please can you explain why all this is changing? The commit comment says "Update footer partial" which doesn't give many clues

Reply:
There is a comment above the first `param.links.bottom` and other comments above that explain.

Basically the `footer` partial of Docsy has been split into 3 partials, one for each column (left, right, center). By default in Docsy the "user" links are shown in the left column and the "developer" links are shown in the right column. Until now we changed the whole partial to show "developer" links in the center, instead of on the right. Now, to minimize the changes compared to the upstream partials, I created a new type of links, "bottom", so we can show them in the center by changing only the center partial.

Reply:
Ok, I get it now. It would have been nice if that explanation had made it into a commit comment or something, to help with review/future reference.